index.html

```diff
@@ -24,8 +24,127 @@ <h1>I’m Mariane Glad to see you!</h1>
             <li class="social" ><a href=""><img src="assets/Vector2.png" alt=""></a></li>
             <li class="social" ><a href=""><img src="assets/Vector3.png" alt=""></a></li>
         </ul>
-        
     </div>
     </section>
+    <section class="works">
+        <div class="cards">
+            <img src="./assets/Snapshoot_Portfolio.png" alt="">
+            <div class="card1">
+            <h2>Tonic</h2>
```

Comment:
- [ ] You did an excellent job on the styles but for this project, you are supposed to stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)design as much as possible. Please increase the font weight for all the card titles and make sure it matches the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/185661312-95c0288a-8067-4c20-a022-62c49731c372.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/185661278-f5b5b17e-446d-4b39-b6cf-2cb9fec11c56.png)